pkg/cmd/search/repos/repos_test.go

```diff
@@ -188,7 +188,7 @@ func TestReposRun(t *testing.T) {
 				},
 			},
 			tty:        true,
-			wantStdout: "\nShowing 3 of 300 repositories\n\ntest/cli     of course  private, archived  Feb 28, 2021\ntest/cliing  wow        public, fork       Feb 28, 2021\ncli/cli      so much    internal           Feb 28, 2021\n",
```

Comment:
Interesting how in tty mode, these timestamps switched from being absolute to being relative `about 1 year ago`?

Reply:
Yeah in tty mode these timestamps changed from using `FuzzyAgoAbbr` to `FuzzyAgo`.